debian/rules

```diff
@@ -65,7 +66,6 @@ CONFFLAGS_deb += \
 	--enable-coredump \
 	--enable-elfutils \
 	--disable-apparmor \
-	--enable-libcurl \
 	--enable-importd \
```

Comment:
I searched for `HAVE_LIBCURL` and its other use is as part of `ENABLE_IMPORTD` in `Makefile.am`, along with a number of other bits:

```
if ENABLE_IMPORTD

if HAVE_LIBCURL
if HAVE_XZ
if HAVE_ZLIB
if HAVE_BZIP2
if HAVE_GCRYPT

…
```

It appears that we don't currently include `systemd-importd` in our images. Looking at `Build-Depends`, I think it's because xz is missing.

Should we drop libcurl from `Build-Depends`? Should we explicitly `--disable-importd` for clarity?